tests/e2e/rbac_privileges_test.go

```diff
@@ -195,17 +176,12 @@ var _ = Describe("RBAC Privileges", func() {
 				{
 					APIGroups: []string{""},
 					Resources: []string{"secrets"},
-					Verbs:     []string{"list", "watch", "get", "create", "delete", "update"},
+					Verbs:     []string{"watch", "list", "get", "create", "update", "delete"},
 				},
 				{
 					APIGroups: []string{"cert-manager.io"},
 					Resources: []string{"certificates"},
-					Verbs:     []string{"patch", "list", "watch", "get", "create", "delete"},
-				},
-				{
-					APIGroups: []string{"cert-manager.io"},
-					Resources: []string{"issuers"},
-					Verbs:     []string{"list", "watch"},
+					Verbs:     []string{"watch", "list", "get", "create", "patch", "delete"},
 				},
 			}
 			Expect(GetRoleBindingRolePolicyRules(ctx,
```

Comment:
Can we change the list asserts in this e2e test from `.To(Equal(*))` to `.Should(ConsistOf(*))`?
So for `crdRoleRules`, `leaderElectionRoleRules`, `klmManagerCertManagerRoleRules`, etc.

This should eliminate the need for items to be in the same order, just checking item equality.